engine/chain/models/bhs_error.go

```diff
@@ -0,0 +1,7 @@
+package chainmodels
+
+// BHSError is an error response that is returned from Block Header Service (BHS)
+type BHSError struct {
```

Comment:
is it expose anywhere by a service, maybe it could be private and in the bhs package?

Reply:
moved